pressbooks/pressbooks
# fix: wrap textbox content in `p` tags

Issue #3213

This PR aims to fix a bug where standard texboxes don't add a `p` tag around the content. This change prevents one from leaving the textbox area when pressing enter while writing the content, keeping the same behaviour as the Educational textboses.

**How to test**

1. Edit a book chapter
2. Choose to insert one of the following textboses: Standard, Standard (sidebar), Shaded, Shaded (sidebar)
3. Make sure the textbox content is wrapped with a `p` tag
4. When pressing enter you should be able to continue editing inside the textbox in a new line
